src/Cli/InstallPluginCommand.php

```diff
@@ -33,12 +33,22 @@
             'message' => 'Grants access to given sections to specified administrator',
             'parameters' => [
                 'roleName' => 'Configurator',
-                'sections' => ['configuration', 'rbac'],
+                // based on config.yml file and added during command's execution
+                'sections' => [],
             ],
             'interactive' => true,
         ],
     ];
 
+    /** @var array */
+    private $rbacConfiguration;
+
+    public function __construct(array $rbacConfiguration)
+    {
+        parent::__construct();
```

Comment:
Not needed

Reply:
After `parent::__construct()`  removal the command does not work properly - CLI says that the command is not configured :(